src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
@bhouston 
As long as all characters are different there shouldn't be a problem using multiple mixers. 

When you have many instances of a single character / object (e.g. mobs in an FPS game) we'd currently need one mixer to handle all of them for reasonable resource consumption.

The potential name conflicts are worth resolving in any case, I think.